Mage.Sets/src/mage/sets/dissension/PsychicPossession.java

```diff
@@ -0,0 +1,125 @@
+ *  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ *  The views and conclusions contained in the software and documentation are those of the
+ *  authors and should not be interpreted as representing official policies, either expressed
+ *  or implied, of BetaSteward_at_googlemail.com.
+ */
+package mage.sets.dissension;
+
+import java.util.UUID;
+import mage.abilities.TriggeredAbilityImpl;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.effects.common.AttachEffect;
+import mage.abilities.effects.common.DrawCardSourceControllerEffect;
+import mage.abilities.effects.common.SkipDrawStepEffect;
+import mage.abilities.keyword.EnchantAbility;
+import mage.cards.CardImpl;
+import mage.constants.CardType;
+import mage.constants.Outcome;
+import mage.constants.Rarity;
+import mage.constants.Zone;
+import mage.game.Game;
+import mage.game.events.GameEvent.EventType;
+import mage.game.events.GameEvent;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.TargetPlayer;
+import mage.target.common.TargetOpponent;
+import mage.target.targetpointer.FixedTarget;
+
+/**
+ *
+ * @author spjspj
+ */
+public class PsychicPossession extends CardImpl {
+
+    public PsychicPossession(UUID ownerId) {
+        super(ownerId, 30, "Psychic Possession", Rarity.RARE, new CardType[]{CardType.ENCHANTMENT}, "{2}{U}{U}");
+        this.expansionSetCode = "DIS";
+        this.subtype.add("Aura");
+
+        // Enchant opponent
+        TargetPlayer auraTarget = new TargetOpponent();
+        this.getSpellAbility().addTarget(auraTarget);
+        this.getSpellAbility().addEffect(new AttachEffect(Outcome.DrawCard));
+        this.addAbility(new EnchantAbility(auraTarget.getTargetName()));
+
+        // Skip your draw step.
+        this.addAbility(new SimpleStaticAbility(Zone.BATTLEFIELD, new SkipDrawStepEffect()));
+
+        // Whenever enchanted opponent draws a card, you may draw a card.
+        this.addAbility(new PsychicPossessionTriggeredAbility());
+    }
+
+    public PsychicPossession(final PsychicPossession card) {
+        super(card);
+    }
+
+    @Override
+    public PsychicPossession copy() {
+        return new PsychicPossession(this);
+    }
+}
+
+class PsychicPossessionTriggeredAbility extends TriggeredAbilityImpl {
+
+    public PsychicPossessionTriggeredAbility() {
+        super(Zone.BATTLEFIELD, new DrawCardSourceControllerEffect(1), true);
+    }
+
+    public PsychicPossessionTriggeredAbility(final PsychicPossessionTriggeredAbility ability) {
+        super(ability);
+    }
+
+    @Override
+    public PsychicPossessionTriggeredAbility copy() {
+        return new PsychicPossessionTriggeredAbility(this);
+    }
+
+    @Override
+    public boolean checkEventType(GameEvent event, Game game) {
+        if (event.getType() == EventType.DRAW_CARD || event.getType() == EventType.DREW_CARD) {
+            return event.getType() == EventType.DREW_CARD;
+        }
```

Comment:
Oh yeah, forgot to take that out ;)  Just had it there for debug purposes and neglected to remove it later on upload.